docs/datetime.txt

```diff
@@ -201,25 +201,24 @@ However, in some applications it is important that
 return different times, as they would normally if the application is
 run anywhere other than the UTC timezone.
 
-The best way to understand how to use
-:func:`~testfixtures.test_datetime` in these situations is to think of
-the internal queue as being a queue of :class:`~datetime.datetime`
-objects at the current local time with a `tzinfo` of None, much as
-would be returned by :meth:`~datetime.datetime.now`. 
-If you pass in a `tz` parameter to
+For these situations, you can explicitly specify what timezone the
+internal queue of :class:`~datetime.datetime` objects should be
+interpreted in by passing in a `tzinfo` to
+:func:`~testfixtures.test_datetime`. (When not specified, this
+defaults to UTC -- which is why both :meth:`~tdatetime.now` and
+:meth:`~tdatetime.utcnow` return the same values in the simple case
+where a `tzinfo` is not specified.)
+
+In this case, :meth:`~tdatetime.now` will return the internal time as
+a naive time in the local timezone, and :meth:`~tdatetime.utcnow` will
+first convert the internal time to UTC, and then return that as a naive
+time. Just as the corresponding :class:`~datetime.datetime` methods do.
```

Comment:
I'm not sure that's what `datetime.datetime` methods do, I think `utcnow` just has an entirely difference codepath, you'd need to check.

Reply:
I don't need to check the code? Are we comfortable that datetime.datetime.now() returns the current local time, and datetime.datetime.utcnow() returns the current time in UTC?

Reply:
I'm never comfortable when it comes to timezones, moreso with the level of indirection that comes from mocking them out ;-) 

https://docs.python.org/3/library/datetime.html#datetime.datetime.now is where we should start, and it's subtly different from your assertion depending on whether `tz` is passed or not:

no `tz`: Return the current local date and time.

`tz`: "the current date and time are converted to tz’s time zone" - I'm actually unsure what this means, does this mean the numbers in the date and time are left as they are and a tz is tacked on the end? Or does it mean the numbers are munged by the tz and then the new numbers + tz are returned? If the latter, what timezone are the original numbers assumed to be in?

Interesting to note the big red box in the `utcnow()` section...

Reply:
To be clear, I was only talking about the call to now() with no arguments.

But since you asked about `now(tz)`:
* _I'm actually unsure what this means_ -- the word 'convert' means 'convert from one timezone to another'. As in, take a London local time and _convert_ that to the equivalent time in New York.
* The alternative is _replacement_ of the tzinfo without any time _conversion_ and is achieved via `datetime.replace(tzinfo=...)`.
* _what timezone are the original numbers assumed to be in?_ -- well, the "original numbers" are "the current local time" so that is interpreted as being a local time, in the system's local timezone.